pkg/jx/cmd/common_install.go

```diff
@@ -1694,7 +1694,11 @@ func (o *CommonOptions) isProw() (bool, error) {
 	if ns == "" {
 		ns = o.currentNamespace
 	}
-	env, err := kube.GetEnvironment(o.jxClient, ns, "dev")
+	jxClient, _, err := o.JXClient()
```

Comment:
You can configure the jx client and set the dev environment in one call with `JXClientAndDevNamespace`.

Reply:
I've fixed this